core/store/src/test/java/uk/gov/gchq/gaffer/store/StoreTest.java

```diff
@@ -256,32 +249,6 @@ public void after() {
         JSONSerialiser.update();
     }
 
-    @Test
-    public void shouldExecuteOperationWhenJobTrackerCacheIsBroken(@Mock final StoreProperties storeProperties) throws Exception {
-        // Given
-        ICache<Object, Object> mockICache = Mockito.mock(ICache.class);
-        doThrow(new CacheOperationException("Stubbed class")).when(mockICache).put(any(), any());
-        ICacheService mockICacheService = Mockito.spy(ICacheService.class);
-        given(mockICacheService.getCache(any())).willReturn(mockICache);
-
-        Field field = CacheServiceLoader.class.getDeclaredField("SERVICES");
-        field.setAccessible(true);
-        java.util.Map<String, ICacheService> mockCacheServices = (java.util.Map<String, ICacheService>) field.get(new HashMap<>());
-        mockCacheServices.put(JOB_TRACKER_CACHE_SERVICE_NAME, mockICacheService);
-
-        final AddElements addElements = new AddElements();
-        final StoreImpl3 store = new StoreImpl3();
-        store.initialise("graphId", createSchemaMock(), storeProperties);
-
-        // When
-        store.execute(addElements, context);
```

Comment:
This test is worth keeping as it tests job functionality
We no longer handle jobs in the `execute` method, but we do in the `executeJob` method.  
Swapping this line should pass the test